.github/workflows/image-publish-on-push-to-branch.yml

```diff
@@ -11,23 +11,31 @@ concurrency:
 
 jobs:
   codeql_analyze:
-    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-codeql.yaml@1.2.0
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-codeql.yaml@1.3.1
     permissions:
       actions: read
       contents: read
       security-events: write
   nest_lint:
-    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-nest-lint.yaml@1.2.0
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-nest-lint.yaml@1.3.1
     with:
       node_version: '18'
     permissions:
       contents: read
+  tests_and_sonarcloud:
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-nest-test-sonarcloud.yaml@DBP-332-sonarcloud
```

Comment:
Change when https://github.com/dBildungsplattform/dbp-github-workflows/pull/6 is merged and version tag added.